alacritty/src/main.rs

```diff
@@ -59,9 +60,11 @@ use crate::config::monitor::Monitor;
 use crate::config::Config;
 use crate::display::Display;
 use crate::event::{Event, EventProxy, Processor};
+use crate::message_bar::MessageBuffer;
 
 fn main() {
-    panic::attach_handler();
+    #[cfg(windows)]
+    crate::panic::attach_handler();
```

Comment:
Do we want to import `crate::panic`? Accessing things with the full path directly using `crate::` seems very rare in our codebase.